planet/cli/subscriptions.py

```diff
@@ -559,16 +554,13 @@ def request_catalog(item_types,
               type=types.DateTime(),
               help='Date and time to end subscription.')
 @pretty
-def request_pv(var_type, var_id, geometry, start_time, end_time, pretty):
+def request_pv(var_id, geometry, start_time, end_time, pretty):
```

Comment:
full support for removing source_type as required input here. 

however! 3.x is an opportunity to better align the SDK with how these products are currently described in the API.

this method is also how one can create an ARPS subscription, which is not a PV.

in the docs [1] [2] we refer to these as "subscription sources" - not the most descriptive name ever but at least it is not coupled to a subset of the products available here.

what do you think about making that change as well? not necessarily as part of this MR, of course.

[1] https://docs.planet.com/develop/apis/subscriptions/sources/#planetary-variable-and-analysis-ready-source-types
[2] https://docs.planet.com/develop/apis/subscriptions/reference/#tag/subscriptions/operation/createSubscription